Avoid newer setuptools using a dir like "pwquality-$V-py3.9-openbsd-7.2-amd64 "

Index: python/Makefile.am
--- python/Makefile.am.orig
+++ python/Makefile.am
@@ -14,7 +14,7 @@ all-local:
 	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV)
 
 install-exec-local:
-	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV) install --prefix=${DESTDIR}${prefix}
+	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV) install --root=${DESTDIR} --single-version-externally-managed
 
 clean-local:
 	rm -rf py$(PYTHONREV)
